src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx

```diff
@@ -18,11 +18,21 @@ import { useAppSelector } from 'state/store'
 
 import { TwirlyToggle } from '../TwirlyToggle'
 
-export type TradeSuccessProps = { handleBack: () => void; children: JSX.Element }
+export type TradeSuccessProps = {
+  handleBack: () => void
+  children: JSX.Element
+  titleTranslation?: string
+  descriptionTranslation?: string
```

Comment:
```suggestion
  titleTranslation?: string | [string, InterpolationOptions]
  descriptionTranslation?: string | [string, InterpolationOptions]
```

Reply:
Done in [3d19513](https://github.com/shapeshift/web/pull/7992/commits/3d1951386bf5e0dba41a1d9082f2a0598502f0ae)